src/user-guide/writing-workflows/jinja2.rst

```diff
@@ -239,13 +241,23 @@ Here's the result:
 
 .. _jinja2-template-variables:
 
-Jinja2 Default Values And Template Inputs
------------------------------------------
+Jinja2 Default Values And Template Variables
+--------------------------------------------
 
-The values of Jinja2 variables can be passed in from the Cylc command
-line rather than hardwired in the workflow configuration.
+You can provide template variables to Cylc in 3 ways:
 
-This can be done on a case-by-case basis using the ``-s`` option e.g:
+- Using the ``--set-file`` option.
+- Using the ``-s`` option.
+- Using a plugin, such as :ref:`Cylc Rose`.
```

Comment:
Would be nice if these could link to the sections below. But not essential

Reply:
```suggestion
- :ref:`Using a plugin`, such as :ref:`Cylc Rose`.
```